src/Microsoft.PowerShell.Commands.Management/commands/management/TestConnectionCommand.cs

```diff
@@ -1,8 +1,12 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
@vexx32, your last commit had 6 failures in `PowerShell-CI-linux`
(These are 5 of the failures)

Test-Connection.Ping.Force IPv4 with explicit PingOptions
```powershell
Expected $true, but got $null.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Test-Connection.Tests.ps1: line 98
98:                 $result1[0].Reply.Options.DontFragment | Should -BeTrue
```
Breakpoint SDK Unit Tests.Managing breakpoints in a remote runspace via the SDK.Can disable breakpoints
```powershell
Exception calling "DisableBreakpoint" with "1" argument(s): "The background process reported an error with the following message: Remote transport error: ."
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/SDK/Breakpoint.Tests.ps1: line 99
99:                 $bp = $jobRunspace.Debugger.DisableBreakpoint($bp)                

```
Breakpoint SDK Unit Tests.Managing breakpoints in a remote runspace via the SDK.Can enable breakpoints
```powershell
Exception calling "GetBreakpoints" with "0" argument(s): "The runspace state is not valid for this operation."
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/SDK/Breakpoint.Tests.ps1: line 105
105:             foreach ($bp in $jobRunspace.Debugger.GetBreakpoints()) {

```
Breakpoint SDK Unit Tests.Managing breakpoints in a remote runspace via the SDK.Can remove breakpoints
```powershell
Exception calling "GetBreakpoints" with "0" argument(s): "The runspace state is not valid for this operation."
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/SDK/Breakpoint.Tests.ps1: line 120
120:             foreach ($bp in $jobRunspace.Debugger.GetBreakpoints()) {

```
Breakpoint SDK Unit Tests.Managing breakpoints in a remote runspace via the SDK.Returns an empty collection when there are no breakpoints
```powershell
Exception calling "GetBreakpoints" with "0" argument(s): "The runspace state is not valid for this operation."
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/SDK/Breakpoint.Tests.ps1: line 126
126:             $jobRunspace.Debugger.GetBreakpoints() | Should -HaveCount 0

```